lib/redbird/plug/session/redis.ex

```diff
@@ -25,17 +24,24 @@ defmodule Plug.Session.REDIS do
   end
 
   def put(conn, nil, data, init_options) do
-    put(conn, Key.generate(), data, init_options)
+    key =
+      Key.generate()
+      |> Key.sign_key(conn)
```

Comment:
Ah, yes. This is a good catch 👍

I think this is where we're supposed to sign the key since it's the first time it's generated.